kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/Order.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.good.patterns.food2door;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Order {
+    Supplier supplier;
```

Comment:
order should contain product, price and user who ordered that. Then supplier of given product should not be placed in order and to get it you need to call another service. I can imagine that somewhere you have method getSupplierOf(givenProduct) which returns you supplier of given product